indico/modules/categories/client/js/components/CalendarLegend.jsx

```diff
@@ -0,0 +1,90 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import './CalendarLegend.module.scss';
+import {Checkbox, Select} from 'semantic-ui-react';
+
+import {Translate} from 'indico/react/i18n';
+
+function LegendItem({title, color, textColor, checked, url, isSpecial, onChange}) {
+  return (
+    <div styleName="legend-item">
```

Comment:
This whole thing has a pointer cursor and feels clickable, but you have to really click the checkbox itself to toggle it...

Reply:
I removed the hover from the CSS.

Reply:
Having to click the checkbox itself is still poor UX, regardless of the hover effect... :/

Reply:
I managed to handle onClick events over the entire div, so now it should be more usable.